### PR TITLE
Update docs for openapi

### PR DESCRIPTION
**Pull request by loay**
connect to https://github.com/strongloop/loopback-next/issues/260

cc @bajtos @raymondfeng @ritch @superkhau

### CODE REVIEW

**Comment by bajtos** [comment]
These to packages were the easy task as they had already some content written 😄

**Comment by crandmck** [comment]
I have some small edits to the other README text, but that text is not part of this PR, which is straightforward/trivial.

File: packages/openapi-spec/README.md [hunk 1]
```diff
@@ -37,6 +37,14 @@ of OpenAPI Specification.
 
 IBM/StrongLoop is an active supporter of open source and welcomes contributions to our projects as well as those of the Node.js community in general. For more information on how to contribute please refer to the [Contribution Guide](https://loopback.io/doc/en/contrib/index.html).
 
-## License
+# Tests
+
+run `npm test` from the root folder.
```
Comment (superkhau) [inline packages/openapi-spec/README.md]: Capital R

File: packages/openapi-spec-builder/README.md [hunk 2]
```diff
@@ -69,6 +69,14 @@ of OpenAPI Specification.
 
 IBM/StrongLoop is an active supporter of open source and welcomes contributions to our projects as well as those of the Node.js community in general. For more information on how to contribute please refer to the [Contribution Guide](https://loopback.io/doc/en/contrib/index.html).
 
-## License
+# Tests
+
+run `npm test` from the root folder.
```
Comment (superkhau) [inline packages/openapi-spec-builder/README.md]: Capital R

Comment (superkhau) [inline packages/openapi-spec-builder/README.md]: other nit is folder -> 'directory' (unless we're in windows-land)

File: packages/openapi-spec/README.md (hunk 1 above)
Comment (superkhau) [inline packages/openapi-spec/README.md]: ditto on windows-land

**Pull request merged**